node/impl/full/eth.go

```diff
@@ -0,0 +1,206 @@
+	// The lotus node is not expected to hold manage accounts, so we'll always return an empty array
+	return []api.EthAddress{}, nil
+}
+
+func (a *EthModule) EthGetBlockTransactionCountByNumber(ctx context.Context, blkNumHex string) (api.EthInt, error) {
+	blkNum, err := strconv.ParseInt(strings.Replace(blkNumHex, "0x", "", -1), 16, 64)
+	if err != nil {
+		return api.EthInt(0), xerrors.Errorf("invalid block number %s: %w", blkNumHex, err)
+	}
+
+	ts, err := a.Chain.GetTipsetByHeight(ctx, abi.ChainEpoch(blkNum), nil, false)
+	if err != nil {
+		return api.EthInt(0), xerrors.Errorf("error loading tipset %s: %w", ts, err)
+	}
+
+	blkMsgs, err := a.Chain.BlockMsgsForTipset(ctx, ts)
+	if err != nil {
+		return api.EthInt(0), xerrors.Errorf("error loading messages for tipset: %v: %w", ts, err)
+	}
+
+	count := 0
+	for _, blkMsg := range blkMsgs {
+		count += len(blkMsg.BlsMessages) + len(blkMsg.SecpkMessages)
+	}
+	return api.EthInt(count), nil
+}
+
+func (a *EthModule) EthGetBlockTransactionCountByHash(ctx context.Context, blkHash string) (api.EthInt, error) {
+	hash, err := api.EthHashFromHex(blkHash)
+	if err != nil {
+		return api.EthInt(0), xerrors.Errorf("invalid hash %s: %w", blkHash, err)
+	}
+
+	ts, err := a.Chain.GetTipSetByCid(ctx, hash.ToCid())
+	if err != nil {
+		return api.EthInt(0), xerrors.Errorf("error loading tipset %s: %w", ts, err)
+	}
+
+	blkMsgs, err := a.Chain.BlockMsgsForTipset(ctx, ts)
+	if err != nil {
+		return api.EthInt(0), xerrors.Errorf("error loading messages for tipset: %v: %w", ts, err)
+	}
+
+	count := 0
+	for _, blkMsg := range blkMsgs {
+		count += len(blkMsg.BlsMessages) + len(blkMsg.SecpkMessages)
+	}
+	return api.EthInt(count), nil
+}
+
+func (a *EthModule) EthGetBlockByHash(ctx context.Context, blkHash string, fullTxInfo bool) (api.EthBlock, error) {
+	return api.EthBlock{}, nil
+}
+
+func (a *EthModule) EthGetBlockByNumber(ctx context.Context, blkNumHex string, fullTxInfo bool) (api.EthBlock, error) {
+	return api.EthBlock{}, nil
+}
+
+func (a *EthModule) EthGetTransactionByHash(ctx context.Context, txHash string) (api.EthTx, error) {
+	return api.EthTx{}, nil
+}
+
+func (a *EthModule) EthGetTransactionCount(ctx context.Context, sender string, blkParam string) (api.EthInt, error) {
+	return api.EthInt(0), nil
+}
+
+func (a *EthModule) EthGetTransactionReceipt(ctx context.Context, blkHash string) (api.EthTxReceipt, error) {
+	return api.EthTxReceipt{}, nil
+}
+
+func (a *EthModule) EthGetTransactionByBlockHashAndIndex(ctx context.Context, blkHash string, txIndexHex string) (api.EthTx, error) {
+	return api.EthTx{}, nil
+}
+
+func (a *EthModule) EthGetTransactionByBlockNumberAndIndex(ctx context.Context, blkNumHex string, txIndexHex string) (api.EthTx, error) {
+	return api.EthTx{}, nil
+}
+
+// EthGetCode returns string value of the compiled bytecode
+func (a *EthModule) EthGetCode(ctx context.Context, address string) (string, error) {
+	return "", nil
+}
```

Comment:
Easy way to get this is to perform the equivalent of an eth_call to the GetBytecode method upcoming here: https://github.com/filecoin-project/ref-fvm/issues/838